scripts/test

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+set -e
+
+if [[ -n ${RASTER_VISION_DEBUG} ]]; then
+    set -x
+fi
+
+function usage() {
+    echo -n \
+        "Usage: $(basename "$0")
+Run linters and tests.
+"
+}
+
+if [[ ${BASH_SOURCE[0]} == "${0}" ]]; then
+    if [[ ${1:-} == "--help" ]]; then
+        usage
+    else
+        if [[ -n ${CI} ]]; then
+            # Execute test suites
+            docker run \
+                --rm -t \
+                "raster-vision-${IMAGE_TYPE}" \
+                /opt/src/scripts/style_tests
+            docker run \
+                -w "$(pwd)" \
+                -v "$(pwd):$(pwd)" \
+                --rm -t \
+                "raster-vision-${IMAGE_TYPE}" \
+                /opt/src/scripts/unit_tests
+            docker run \
+                --rm -t \
+                "raster-vision-${IMAGE_TYPE}" \
+                /opt/src/scripts/integration_tests
+
+            # Create new coverage reports
+            docker run \
+                -w "$(pwd)" \
+                -v "$(pwd):$(pwd)" \
+                --rm -t \
+                "raster-vision-${IMAGE_TYPE}" \
+                coverage xml
```

Comment:
Ditto. I think this stuff would be a lot more clear if we defined the working directories, volume mounts, TTY allocation, etc. in a `docker-compose.yml` file.

Reply:
I agree! I've thought about this a little more and I'd like to do this in a separate PR. We'd need to touch some other scripts like `docker/run` and `docker/build` if we want to avoid duplicating config, and we'd also need to introduce another development dependency (at least until Compose v2 is out of beta and is available with any recent version of Docker Engine). That's going to increase the diff a good bit and will require another more thorough round of local testing to make sure I'm not breaking anybody's workflow. That's something I'd like to make very visible as a separate decision from this migration.

I can get a follow-up issue set up prior to merging. What do you think?

Reply:
Yeah, I'm fine with a follow up issue that outlines concerns.

Reply:
Tracking in #1244.